buildout-build-redhat-64bit-with-libvirt.cfg

```diff
@@ -7,11 +7,23 @@ parts = zlib ncurses readline openssl openssh libgpg-error libgcrypt gettext lib
    gnu
 recipe = hexagonit.recipe.cmmi
 version = 1.02.28
-configure-options = --prefix=${options:prefix} --disable-rpath --libdir=${options:prefix}/lib64 --disable-selinux
+configure-options = --prefix=${options:prefix} --disable-rpath --libdir=${options:prefix}/lib64 --disable-selinux --with-user=$USER --with-group=$GROUP
+url = ${urls:source}/${:name}-${:version}.tgz
```

Comment:
The correct url should be with `.tar.gz`. There was a sync problem between our external and internal source repositories. Our internal repository contained `.tar.gz` archives while the external incorrectly had only `tgz` archives for `libvirt` and `device-mapper`. I fixed this now, and the files now use `tar.gz`, so you should remove the `url` definitions.

Reply:
Good to know about this one, thanks!